### DISCUSSION TITLE
Prisma Error

### DISCUSSION THREAD

**Discussion by sanju-cmd** [prisma/prisma#13729] [Q&A]
Unhandled Runtime Error
Error: PrismaClient is unable to be run in the browser.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues

**Comment by psikoi** [comment]
Seems like it's been discussed before:

https://github.com/prisma/prisma/issues/6219 

That issue might a good place to start.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
